R/get_message.R

```diff
@@ -0,0 +1,40 @@
+# get_message ------------------------------------------------------------------
+get_message <- function(message_id, ...) {
```

Comment:
any reason for this helper function to live in a separate file?

Reply:
Yes. If any other function defined in another file is giving messages or outputs it can define its messages also here. The idea is to define all text messages here. And to fetch them where needed with `get_message()`. This keeps the functions giving the messages clean given that you find good keywords.

Reply:
👌